### DISCUSSION TITLE
A No Nonsense Guide on how to use an M-Series Mac GPU with PyTorch – ForBo7 // Salman Naqvi

### DISCUSSION THREAD

**Discussion by giscus** [ForBo7/forbo7.github.io#16] [Giscus Comments]
# A No Nonsense Guide on how to use an M-Series Mac GPU with PyTorch – ForBo7 // Salman Naqvi

Squeezing out that extra performance.

https://forbo7.github.io/forblog/posts/8_how_to_use_apple_gpu_with_pytorch.html

**Comment by RuizSerra** [comment]
You could avoid the "annoying part" with [`torch.set_default_device()`](https://pytorch.org/docs/stable/generated/torch.set_default_device.html)!

**Reply by ForBo7** [reply]
Oh, thanks for sharing! That's good to know! Yes, that definitely remedies the "annoying part"! 😄
